### DISCUSSION TITLE
Add caption

### DISCUSSION THREAD

**Discussion by udobehm** [ramensoftware/windhawk-mods#76] [General]
Hi,

could you add program captions as they used to be under the old Win 10 taskbar? Like this:

![image](https://user-images.githubusercontent.com/50866188/210178058-2f526de6-d345-4499-a2ca-997db2895892.png)

Could you also add features that the ExplorerPatcher app offers?

Like customizing icons in the system tray:

![image](https://user-images.githubusercontent.com/50866188/210178123-c281a490-f42c-4fd5-ba1a-a9d879cc2bee.png)
![image](https://user-images.githubusercontent.com/50866188/210178189-62e3a4e9-c9bc-4bcb-a8c2-4b93b40cfd7b.png)

Add the ability to disable the Win 11 context menu in the file explorer and always show the old Win 10 menu instead??

Thanks!
Udo

**Comment by m417z** [comment]
> could you add program captions as they used to be under the old Win 10 taskbar?

Have you tried the [Taskbar Labels for Windows 11](https://windhawk.net/mods/taskbar-labels) mod?

> customizing icons in the system tray

Maybe in the future.

> the ability to disable the Win 11 context menu in the file explorer and always show the old Win 10 menu instead?

Maybe in the future, too. You can [vote for it here](https://ramensoftware.com/windhawk-mods-for-the-windows-11-taskbar) under "Right click: Standard window menu".

**Comment by udobehm** [comment]
Thanks for leeting me know.

Have tried this.

Why do pinned programs become opened in a seperate item on the taskbar?

![image](https://user-images.githubusercontent.com/50866188/210180242-03fb1325-d741-45d5-bd6e-9a43d8d20e09.png)

Can this be fixed, so that the programs are opening where the pinned items are?

**Reply by m417z** [reply]
You're probably also using the Disable Grouping mod. It's a known limitation: https://github.com/ramensoftware/windhawk-mods/issues/48

**Comment by udobehm** [comment]
So this can or will not be fixed?

**Reply by m417z** [reply]
That's not easy to fix. I might look at it in the future, but I don't have a timeframe.

**Comment by udobehm** [comment]
Could you write me as soon as it's working?

**Reply by m417z** [reply]
You can subscribe to the issue, go to the link I posted and you'll find a "Subscribe" button on the right.

**Comment by udobehm** [comment]
Ok, thanks!

**Comment by udobehm** [comment]
Hi, is it working now?

**Reply by m417z** [reply]
No, there's an open issue for it.
